### DISCUSSION TITLE
[UX] Remember selected time in photos-section

### DISCUSSION THREAD

**Discussion by Micha10** [immich-app/immich#13153] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Comming back to the photos-section always starts at the beginning and the user has to scroll to the desired time. It's a little bit anoying after the 45th time.

For example: You browse a picture at 1980th and want to see the location on the map. You click at the right bottom the icon for opening the map. Immich is routing to the map section and there's no easy way to go back to the picture. After clicking "photos" you start at the beginning and have to browse all the way down (even if you try to click at the correct year at the right bar) and scroll over possible hundreds of photos to find the last opened one. 

So my wish is: Remember the last opened location and after leaving and coming back select the last viewed area of photos.

As an alternative: Regardless where you are, there could be an icon/button "Goto the current photo in the timeline"

Thanks for that great software

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3114
